README.md

```diff
@@ -97,60 +98,26 @@ interface for the plugin. After filling the entries and click "Generate Pipeline
 sample scripts which can be used in your Pipeline definition.
 
 ```groovy
-kubernetesDeploy(
-        credentialsType: 'KubeConfig',
-        kubeConfig: [path: '<path-to-the-kubeconfig-in-the-workspace>'],
+kubernetesDeploy(kubeconfig: 'kubeconfig-credentials-id',                 // REQUIRED
```

Comment:
Is the old syntax still supported?

Reply:
This looks very similar to old syntax, change to something like kubeconfigId?

Reply:
The old syntax is still supported for compatibility concern.

I struggled on the naming actually. Let me update it to be more expressive (`kubeconfigId`) instead of the current simple one.

Reply:
updated to `kubeconfigId`